### DISCUSSION TITLE
迁移gogs

### DISCUSSION THREAD

**Discussion by HarryWang29** [gogs/gogs#6751] [中文讨论] [answered]
之前安装gogs的机器到期了，当时我也草率的只备份了gogs-repositories文件夹

这几天正好有时间在重新搭建gogs，想把之前的gogs-repositories都迁移进来
但是我这边目录都设置完成了，在gogs的管理界面里，还是看不到相应的仓库

这次使用了docker部署，查看路径都显示正常

![image](https://user-images.githubusercontent.com/8288067/151186524-48288ac6-0c61-4c68-812d-3527250423c7.png)

![image](https://user-images.githubusercontent.com/8288067/151186767-732ad866-c2af-4767-bbfe-559def73a21b.png)

![image](https://user-images.githubusercontent.com/8288067/151186979-87b892bd-f55a-40e8-8ac4-7b627dcb0d77.png)

![image](https://user-images.githubusercontent.com/8288067/151187172-7436836e-c71f-421c-9059-cad9f4a9d18d.png)

--------------------------------------------------------------------

我尝试在mysql的`repository`表中增加了一条记录，虽然在gogs页面中，能够看到这个仓库名称，但是看上去，并没有加载本地仓库

**Comment by unknwon** [comment]
因为你没有备份数据库。。兄弟...

**Comment by HarryWang29** [comment, marked as answer]
我知道啊，我是想问怎么办，后来微信群里的@JOE给了方法，先在网页创建同名仓库，然后再把repository覆盖进去，解决了

**Reply by sandianyiyuan** [reply]
机智哈哈

**Reply by xstchan** [reply]
覆盖后出现 [ERROR] [...io/gogs/internal/context/repo.go:249 func1()] get tags: exit status 128 - fatal: detected dubious ownership in repository at
